content/en/docs/components/central-dash/overview.md

```diff
@@ -16,19 +16,27 @@ following features:
 - A housing for the UIs of the components running in the cluster, including
   **Pipelines**, **Katib**, **Notebooks**, and more.
 - A [registration flow](/docs/components/central-dash/registration-flow/) that
-  prompts new users to set up their namespace if necessary. 
+  prompts new users to set up their namespace if necessary.
 
 ## Overview of Kubeflow UIs
 
 The Kubeflow UIs include the following:
 
-* **Home**, a central dashboard for navigation between the Kubeflow components.
-* **Pipelines** for a Kubeflow Pipelines dashboard.
-* **Notebook Servers** for Jupyter notebooks.
-* **Katib** for hyperparameter tuning.
-* **Artifact Store** for tracking of artifact metadata.
-* **Manage Contributors** for sharing user access across namespaces in the 
-  Kubeflow deployment.
+* **Home**:, Home, the central hub to access recent resources, active
+  experiments, and useful documentation.
+* **Notebook Servers**: To manage [Notebook servers](/docs/components/notebooks/).
+* **TensorBoards**: To manage TensorBoard servers.
+* **Models**: To manage deployed [KFServing models](/docs/components/kfserving/kfserving/).
+* **Volumes**: To manage the cluster's Volumes.
+* **Experiments (AutoML)**: To manage [Katib](/docs/components/katib/) experiments.
+* **Experiments (KFP)**: To manage [Kubeflow Pipelines (KFP)](/docs/components/pipelines/) experiments.
+* **Pipelines**: To manage KFP pipelines.
+* **Runs**: To manage KFP runs.
+* **Recurring Runs**: To manage KFP recurring runs.
+* **Artifacts**: To track MLMD artifacts.
```

Comment:
Can we change "MLMD artifacts" to "ML Metadata (MLMD) artifacts of KFP"?

Reply:
I just noticed there are two rows of Artifacts, we should remove the duplicated one here and keep the same order with central dashboard UI

Reply:
Oh you are right, I agree with your suggestion here.